nexus/external-api/src/lib.rs

```diff
@@ -3542,104 +3542,137 @@ pub trait NexusExternalApi {
 
-    /// Create webhook receiver
+    /// Delete alert receiver
     #[endpoint {
-        method = POST,
-        path = "/v1/webhooks/receivers",
-        tags = ["system/webhooks"],
+        method = DELETE,
+        path = "/v1/alert-receivers/{receiver}",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_create(
+    async fn alert_receiver_delete(
         rqctx: RequestContext<Self::Context>,
-        params: TypedBody<params::WebhookCreate>,
-    ) -> Result<HttpResponseCreated<views::WebhookReceiver>, HttpError>;
+        path_params: Path<params::AlertReceiverSelector>,
+    ) -> Result<HttpResponseDeleted, HttpError>;
 
-    /// Update webhook receiver
-    ///
-    /// Note that receiver secrets are NOT added or removed using this endpoint.
-    /// Instead, use the `/v1/webhooks/{secrets}/?receiver={receiver}` endpoint
-    /// to add and remove secrets.
+    /// Add alert receiver subscription
     #[endpoint {
-        method = PUT,
-        path = "/v1/webhooks/receivers/{receiver}",
-        tags = ["system/webhooks"],
+        method = POST,
+        path = "/v1/alert-receivers/{receiver}/subscriptions",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_update(
+    async fn alert_receiver_subscription_add(
         rqctx: RequestContext<Self::Context>,
-        path_params: Path<params::WebhookReceiverSelector>,
-        params: TypedBody<params::WebhookReceiverUpdate>,
-    ) -> Result<HttpResponseUpdatedNoContent, HttpError>;
+        path_params: Path<params::AlertReceiverSelector>,
+        params: TypedBody<params::AlertSubscriptionCreate>,
+    ) -> Result<HttpResponseCreated<views::AlertSubscriptionCreated>, HttpError>;
 
-    /// Delete webhook receiver
+    /// Remove alert receiver subscription
     #[endpoint {
         method = DELETE,
-        path = "/v1/webhooks/receivers/{receiver}",
-        tags = ["system/webhooks"],
+        path = "/v1/alert-receivers/{receiver}/subscriptions/{subscription}",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_delete(
+    async fn alert_receiver_subscription_remove(
         rqctx: RequestContext<Self::Context>,
-        path_params: Path<params::WebhookReceiverSelector>,
+        path_params: Path<params::AlertSubscriptionSelector>,
     ) -> Result<HttpResponseDeleted, HttpError>;
 
-    /// Add webhook receiver subscription
+    /// List delivery attempts to alert receiver
+    ///
+    /// Optional query parameters to this endpoint may be used to filter
+    /// deliveries by state. If none of the `failed`, `pending` or `delivered`
+    /// query parameters are present, all deliveries are returned. If one or
+    /// more of these parameters are provided, only those which are set to
+    /// "true" are included in the response.
+    #[endpoint {
+        method = GET,
+        path = "/v1/alert-deliveries",
+        tags = ["system/alerts"],
+    }]
+    async fn alert_delivery_list(
```

Comment:
we had decided to call this `alert receiver log` in the CLI. It is particular to a specific receiver, right?

Reply:
yes, it's particular to a receiver, which is taken as a query param. i think `log` is a good name for the CLI, but in the API it felt like it should be a "list of delivery resources"...it could also be `/v1/alert-receivers/{name-or-id}/deliveries` or something?

Reply:
> /v1/alert-receivers/{name-or-id}/deliveries

That's what I was thinking.

Reply:
yeah, we should be able to do that. i'll note that the previous route was `/v1/webhook-deliveries?receiver=...` and i think that may also have been at @david-crespo's urging IIRC.

Reply:
Yep, `/v1/alert-receivers/{name-or-id}/deliveries` sounds good to me. I see we already have `/v1/alert-receivers` and subscriptions hanging off of it.

Reply:
Done in eae40e0716a7ea376708fb15c13efe648722d925!